pkg/core/managers/apis/mesh/mesh_manager_test.go

```diff
@@ -87,13 +74,9 @@ var _ = Describe("Mesh Manager", func() {
 		It("should create default resources", func() {
 			// given
 			meshName := "mesh-1"
-			resKey := model.ResourceKey{
-				Name: meshName,
-			}
 
 			// when
-			mesh := core_mesh.NewMeshResource()
-			err := resManager.Create(context.Background(), mesh, store.CreateBy(resKey))
+			err := samples.MeshDefaultBuilder().WithName(meshName).Create(resManager)
```

Comment:
I think we should more aggressively panic in tests to avoid noise but this isn't any worse than before

Reply:
I'd leave an error to open possibility to use `Create` with Eventually